src/main/java/net/pms/configuration/WebRender.java

```diff
@@ -541,21 +542,44 @@ public void setSubLang(String s) {
 		subLang = s;
 	}
 
-	private ArrayList<String[]> push;
+	private final ArrayList<String[]> push;
 
 	public void push(String... args) {
-		push.add(args);
+		if (sse == null || !sse.isOpened() || !sse.sendMessage(gson.toJson(args))) {
+			synchronized (push) {
+				push.add(args);
+			}
+		}
 	}
 
 	public String getPushData() {
 		String json = "{}";
-		if (push.size() > 0) {
-			json = gson.toJson(push);
-			push.clear();
+		synchronized (push) {
+			if (!push.isEmpty()) {
+				json = gson.toJson(push);
+				push.clear();
+			}
 		}
 		return json;
 	}
 
+	private ServerSentEvents sse;
+	public void addServerSentEvents(ServerSentEvents sse) {
+		if (this.sse != null && this.sse.isOpened()) {
+			this.sse.sendMessage(gson.toJson(new String[] {"close", "warn", "Another tab is browsing the interface"}));
```

Comment:
@SurfaceS The message "another tab is browsing the web interface" works but clicking "OK" closes the current tab, that might be unexpected for a user because the text doesn't mention that.

Let's change it to e.g.
`The web interface is open in another tab which can cause problems. We recommend you close this tab" then the button says "Close this tab" and then another button says "Ignore"

Also let's make this translatable because it is user-facing

Reply:
Translatable now.
I removed the `windows.close()` call as it doesn't work on many web browser (modern ones won't allow scripts to close tab).